### ISSUE TITLE
Support devices with FPS > 60

### ISSUE THREAD

**Issue by Almouro** [bamlab/flashlight#135]
**What**

Max FPS is set to 60 in the code
We need to account for devices with a higher refresh rate

**How to implement**

No details yet, to be provided soon

**Comment by MalcolmTomisin** [comment]
Hi @Almouro 

Checking in on this, I have a [POC](https://github.com/bamlab/flashlight/compare/main...MalcolmTomisin:flashlight:feat/support-fps-120) on getting max FPS on Android devices. There are gaps in my knowledge of how Flashlight's internals passes data around. How do you reckon data be passed to the reporting and web app clients from data gotten from the shell.
Here's what I have tried; I assumed data is passed on around by sockets, I have tried emiting data to the webapp but I haven't been successful at that. I would greatly appreciate your insights here. Huge fan of flashlight.

**Comment by Almouro** [comment]
Thanks @MalcolmTomisin! That's very cool!! ♥️

It's weird that it doesn't get reported properly. 
You can try and run `flashlight measure` or `test` locally https://github.com/bamlab/flashlight/blob/main/CONTRIBUTING.md#measure-command

For the `measure` command, data is indeed transmitted via Socket, measures are updated [here](https://github.com/bamlab/flashlight/blob/main/packages/commands/measure/src/server/ServerSocketConnectionApp.tsx#L54)

Alternatively, you can run the undocumented command `node packages/platforms/android/dist/src/commands.js profile --fps` to have live FPS polling the CLI to make sure the measures are correct.

I think the score should get messed up (because we also have an hardcoded value in flashlight-open-source/packages/core/reporter/src/reporting/getScore.ts) but at least the higher FPS should be visible on the graph 🤔

**Comment by MalcolmTomisin** [comment]
Yup I did get messed up scores...I figured out the hardcoded fps contributed to that. I would get right to your recommendations. Thank you!
